drivers/adc/ad469x/ad469x.c

```diff
@@ -309,13 +309,6 @@ int32_t ad469x_set_channel_sequence(struct ad469x_dev *dev,
 		if (ret != SUCCESS)
 			return ret;
 
-		ret = ad469x_spi_write_mask(dev,
```

Comment:
This could be merged with the commit that added the configuration to `ad469x_adv_sequence_set_num_slots()`.